packages/odyssey-react-mui/src/labs/DataView/DataView.tsx

```diff
@@ -67,6 +68,24 @@ const BulkActionsContainer = styled("div")(() => ({
 const AdditionalActionsContainer = styled("div")(() => ({
   display: "flex",
   justifyContent: "flex-end",
+
+  ["&:empty"]: {
```

Comment:
@jordankoschei-okta This doesn't feel necessary. As far as I can tell, it won't render if it's empty. And if it DOES render when it's empty, we should just fix that logic